loris/resolver.py

```diff
@@ -141,7 +141,7 @@ def resolve(self, app, ident, base_uri):
         format_ = self.format_from_ident(ident)
         uri = self.fix_base_uri(base_uri)
         extra = self.get_extra_info(ident, source_fp)
-        return ImageInfo(app, uri, source_fp, format_, extra)
+        return ImageInfo(app, source_fp, format_, extra)
```

Comment:
Since you’re ditching the `uri` argument here, you can skip defining it on L142 (and I think eventually we get to delete the `fix_base_uri` method).

And in the other two constructors in this file.